src/ui/manager.h

```diff
@@ -185,8 +206,14 @@ namespace ui {
     // restack a window if the user clicks on it.
     Widget* m_lockedWindow;
 
+    // Holds fractional increments on scroll wheel to be computed on next
+    // mouse message.
+    static gfx::PointF g_wheelDeltaFraction;
```

Comment:
Can be a non-static member (as there is just one ui::Manager anyway):
```suggestion
    gfx::PointF m_wheelDeltaFraction;
```